code/game/objects/effects/aliens.dm

```diff
@@ -489,29 +489,30 @@
 /obj/effect/xenomorph/xeno_telegraph
 	name = "???"
 	desc = ""
-	icon_state = "xeno_telegraph_red"
+	icon_state = "xeno_telegraph_base"
 	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
 
-/obj/effect/xenomorph/xeno_telegraph/New(loc, ttl = 10)
+/obj/effect/xenomorph/xeno_telegraph/New(loc, ttl = 10, color = null)
 	..(loc)
+	if(color)
+		src.color = color
 	QDEL_IN(src, ttl)
 
 /obj/effect/xenomorph/xeno_telegraph/red
-	icon_state = "xeno_telegraph_red"
+	color = COLOUR_DARK_RED
 
 /obj/effect/xenomorph/xeno_telegraph/brown
-	icon_state = "xeno_telegraph_brown"
+	color = COLOUR_BROWN
 
 /obj/effect/xenomorph/xeno_telegraph/green
-	icon_state = "xeno_telegraph_green"
+	color = COLOUR_GREEN
 
-/obj/effect/xenomorph/xeno_telegraph/brown/abduct_hook
+/obj/effect/xenomorph/xeno_telegraph/abduct_hook
 	icon_state = "xeno_telegraph_abduct_hook_anim"
 
```

Comment:
Either this needs to inherit brown again, or it should set the color to brown. Or is the intention to change its color to none?

Reply:
The icon for this is brown by default so as long as no colour argument is provided it will be brown. I could change the type path back to brown if that would be clearer? Eventually would like to get these all converted to white with colour arguments but need to find a spriter to do all the sprite work.

Reply:
That's fine if this one then isn't using a white sprite to be colored, but then why set the one below this when it didn't used to have a color set to brown (it only inherited a path that was brown in name but that path had a white color)?